src/Agent.Worker/ExecutionContext.cs

```diff
@@ -109,6 +110,7 @@ public IExecutionContext CreateChild(Guid recordId, string name)
 
         public void Start(string currentOperation = null)
         {
+            _secretMasker = HostContext.GetService<ISecretMasker>();
```

Comment:
moved _secretMasker to Initialize, but cannot move _logger.Setup, because _mainTimelineId, _record.Id are not set yet.